app/datasets.js

```diff
@@ -0,0 +1,225 @@
+'use strict';
+
+module.exports = {
+    castArray: function ()
+    {
+        return [];
+    },
+    clone: function ()
+    {
+        return [];
+    },
+    cloneDeep: function (obj)
+    {
+        return [];
+    },
+
+    /*
+     cloneDeepWith :function(){
+     return [];
+     },
+     cloneWith : function(){
+     return [];
+     },
+     */
+
+    eq: function ()
+    {
+        return [];
+    },
+    gt: function ()
+    {
+        return [];
+    },
+    gte: function ()
+    {
+        return [];
+    },
+    isArguments: function ()
+    {
+        return [];
+    },
+    isArray: function ()
+    {
+        return [];
+    },
+    isArrayBuffer: function ()
+    {
+        return [];
+    },
+    isArrayLike: function ()
+    {
+        return [];
+    },
+    isArrayLikeObject: function ()
+    {
+        return [];
+    },
+    isBoolean: function ()
+    {
+        return [];
+    },
+    isBuffer: function ()
+    {
+        return [];
+    },
+    isDate: function ()
+    {
+        return [];
+    },
+
+    /*
+     isElement: function(){
+     return [document.body];
+     },
+     */
+
+    isEmpty: function ()
+    {
+        return [];
+    },
+    isEqual: function ()
+    {
+        return [];
+    },
+    isEqualWith: function ()
+    {
+        return [];
+    },
+    isError: function ()
+    {
+        return [];
+    },
+    isFinite: function ()
+    {
+        return [];
```

Comment:
It should be ok for - Infinity becouse test return false when value isn't finite number.